### DISCUSSION TITLE
Configuring Redis Eviction Policy

### DISCUSSION THREAD

**Discussion by thenithishrajan** [taskforcesh/bullmq#2583] [Q&A]
I received this warning - 'IMPORTANT! Eviction policy is volatile-lru. It should be 'noeviction'.' And I realized that bullmq suggests the same. However, I don't know how to configure this!

This is the code I have:
```
export const redisConnectionOption = {
  port: +process.env.REDIS_PORT!,
  host: process.env.REDIS_HOST,
  reconnectOnError: () => {
    console.log("Redis connection failed... Retrying....");
    return true;
  },
};

const consoleQueue = new Queue("consoleQueue", {
  connection: redisConnectionOption,
  defaultJobOptions: {
    removeOnComplete: true,
    removeOnFail: true,
    attempts: 10,
    backoff: {
      type: "exponential",
      delay: 15000, // start by delaying for 15 sec
    },
  },
});

```

**Comment by manast** [comment]
You configure this setting in Redis. Depending on how are you hosting Redis the way to configure it differs, but basically the setting is read from redis.conf: https://redis.io/docs/latest/develop/reference/eviction/

**Reply by thenithishrajan** [reply]
Thanks!!
